helpers/env_vars.go

```diff
@@ -53,27 +60,88 @@ var (
 	TICSecretEnvVar = "TIC_SECRET"
 )
 
-// EnvVars holds all the environment variable values that a non-test server should have.
+// EnvVars provides a convenient method to access environment variables
 type EnvVars struct {
-	ClientID        string
-	ClientSecret    string
-	Hostname        string
-	LoginURL        string
-	UAAURL          string
-	APIURL          string
-	LogURL          string
-	PProfEnabled    string
-	BuildInfo       string
-	NewRelicLicense string
-	SecureCookies   string
-	LocalCF         string
-	SessionBackend  string
-	SessionKey      string
-	BasePath        string
-	SMTPHost        string
-	SMTPPort        string
-	SMTPUser        string
-	SMTPPass        string
-	SMTPFrom        string
-	TICSecret       string
+	path []EnvLookup
+}
+
+// Get returns value for key if present, else returns defaultVal if not found
+func (el *EnvVars) Get(key, defaultVal string) string {
```

Comment:
@aeijdenberg I was just thinking:

- `String` was a better name than `Get`: so `env.String("FOO", "bar")`
- Then you have `env.Bool("ENABLE_THIS")`
- And `env.MustString("FOO")`

For me `env.String(...)`, `env.Bool(...)` are more intuitive (and consistent) from the outside than `env.Get(...)` and also matches the way the `flag` package does this.

Anyway, just a suggestion.

Reply:
Fixed as suggested.